README.md

```diff
@@ -16,6 +16,18 @@ If you are just including the built files, pixi spine adds itself to a pixi name
 new PIXI.spine.Spine();
 ```
 
+### NPM Install
+
+```bash
+npm install pixi-spine
+```
+
+The correct way to import pixi-spine is to add it to the namespace.
+
+```js
+import 'pixi-spine'
```

Comment:
Thanks for providing instructions! This is throwing and `undefined` error for me unfortunately:

```javascript
Uncaught ReferenceError: PIXI is not defined
```

Importing this way:

```javascript
import * as PIXI from 'pixi.js'
import 'pixi-spine'
```

Could it be something else is required to import this correctly? I'm using Rollup so that could be it.

Reply:
```js
import * as PIXI from 'pixi.js'
window.PIXI = PIXI;
require('pixi-spine');
```

that's because ES6 modules are fucked up beyond repair, its impossible to use them, TS, and vanilla at the same time.

Reply:
oh god yea i have no idea if this will actually work anymore, that pixi project i was working on ended like a year ago, sorry folks

Reply:
No worries, what I find hard about Node and npm is that each module seems to have its own way of being imported or needs to be patched in order to work. 😂 Thanks both!

Reply:
It's because pixi-spine uses PIXI global namespace which defeats the point of it being a "module" anyway.

Reply:
@lloydevans we are almost there! After https://github.com/pixijs/pixi.js/pull/6849 we'll be able to support modules. finally. Currently experimenting it on pixi-tilemap, there still are problems :(

Reply:
@ivanpopelyshev Cool, yea that will be nice. Liking the TS conversion of Pixi also.

Reply:
https://github.com/pixijs/pixi.js/pull/6849 seems to be merged now, any updated on making pixi-spine working with TS correctly?

I'm having almost impossible time including this into Angular 11 project.